docs/fastdds/ros2/discovery_server/ros2_discovery_server.rst

```diff
@@ -310,6 +304,131 @@ partition from *Talker 2* and so it does not listen to it.
     listen to each other messages.
 
 
+ROS 2 Introspection
+-------------------
+
+ROS 2 Command Line Interface (CLI) implements several introspection features to analyze the behaviour of a ROS2
+execution.
+These features (i.e. `rosbag`, `topic list`, etc.) are very helpful to understand a ROS 2 working network.
+
+Most of these features use the DDS capability to share any topic info with every exiting participant.
+However, the new :ref:`ros2_discovery_server_v2` implements a traffic network reduction
+that limits the discovery data between nodes that do not share a topic.
+This means that not every node will receive every topic data unless it has a reader in that topic.
+
+The Discovery Server v2 functionality allows every server to know and share all the participants and topic information
+with every other server matched.
+So, this functionality can be used in order to use ROS 2 introspection successfully when running with Discovery Server
+discovery protocol.
+
+
+Daemon's related commands
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ROS 2 Daemon is used in several ROS 2 CLI introspection commands.
+This section will explain how to use ROS 2 CLI with ROS 2 Daemon running as a Server.
+This will allow the Daemon to know all the Node's graph and to receive every topic and endpoint information.
+
+Fast DDS CLI can execute a Discovery Server, but it will spawn the Server in the actual process (or in a new one),
+and we want to run ROS 2 Daemon process as the Discovery Server.
+For this purpose, Fast DDS XML configuration can be used in order to preconfigure every new node that is created
+with this configuration exported.
```

Comment:
I don't think this is clear enough. Actually, up to here, it is not stated yet that the daemon has its own participant and so on.

Reply:
Please, check if the new additions clarify enough the idea.